### DISCUSSION TITLE
What's the mean of "control_change" Message? And What's the mean of it's parameters "control", "value" and "time"

### DISCUSSION THREAD

**Discussion by ACeYwcy** [mido/mido#463] [Q&A]
What's the mean of "control_change" Message? And What's the mean of it's parameters "control", "value" and "time"

**Comment by rdoursenaud** [comment]
These are MIDI messages to change controllers.

Some are defined by [the standard](https://www.midi.org/specifications-old/item/table-3-control-change-messages-data-bytes-2) but most of the implementation details depends on the sound module or software synthesizer used.

For example the control change 7 changes the volume of the channel.

### Parameters
`control`: The number identifying the controller.
`value`: The value of the controller.
`time`: Depends on the context. Please refer to [the time attribute](https://mido.readthedocs.io/en/latest/messages.html?highlight=time#the-time-attribute) documentation.
